.gitlab-ci.yml

```diff
@@ -15,4 +15,4 @@ libtelio-build-pipeline:
 
   trigger:
     project: $LIBTELIO_BUILD_PROJECT_PATH
-    branch: v1.2.5
+    branch: v1.2.5
```

Comment:
Why these newlines are removed?

Reply:
Oh it is a leftover after conflict resolution. I will fix it.

Reply:
fixed